erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -509,18 +514,17 @@ def set_total_allocated_amount(self):
 		self.base_total_allocated_amount = abs(base_total_allocated_amount)
 
 	def set_unallocated_amount(self):
-		self.unallocated_amount = 0
 		if self.party:
 			total_deductions = sum(flt(d.amount) for d in self.get("deductions"))
 			if self.payment_type == "Receive" \
-				and self.base_total_allocated_amount < self.base_received_amount_after_tax + total_deductions \
-				and self.total_allocated_amount < self.paid_amount_after_tax + (total_deductions / self.source_exchange_rate):
-				self.unallocated_amount = (self.received_amount_after_tax + total_deductions -
+				and self.base_total_allocated_amount < self.base_received_amount + total_deductions \
+				and self.total_allocated_amount < self.paid_amount + (total_deductions / self.source_exchange_rate):
+				self.unallocated_amount = (self.received_amount + total_deductions -
```

Comment:
Ther is error here please fix it line 522 must be:
self.unallocated_amount = (self.base_received_amount + total_deductions -

**base_received_amount**   not **received_amount** 
and it case error when Receive multi currency payment